SourceKitStressTester/Sources/StressTester/SourceKitDocument.swift

```diff
@@ -273,6 +277,41 @@ struct SourceKitDocument {
     return (tree!, response)
   }
 
+  private func checkExpectedCompletionResult(_ expected: ExpectedResult, in response: SourceKitdResponse, info: RequestInfo) throws {
+    let matcher = CompletionMatcher(for: expected)
+    var found = false
+    response.value.getArray(.key_Results).enumerate { (index, item) -> Bool in
+      let result = item.getDictionary()
+      found = matcher.match(result.getString(.key_Name), ignoreArgLabels: shouldIgnoreArgs(of: expected, for: result))
+      return !found
+    }
```

Comment:
Purely stylistic nitpick: I think 
```swift
for (index, item) in response.value.getArray(.key_Results).enumerated() {
  let result = item.getDictionary()
  found = matcher.match(result.getString(.key_Name), ignoreArgLabels: shouldIgnoreArgs(of: expected, for: result))
  if found {
    break
  }
}
```

would read a lot easier.

Reply:
`getArray()` returns a SourceKitdResponse.Array rather than a normal Swift array – it doesn't have `enumerated()` as far as I can tell. The only options SwiftLang gives at the moment are the one here or `for i in 0..<response.value.getArray(.key_Results).count` + index-based access. I can change it to that if you prefer.

Reply:
Ah, I see. You could of course go for 
```swift
Array(response.value.getArray(.key_Results)).enumerated()
```
but you can leave it as is as well. Both is fine with me.

Reply:
SourceKitdResponse.Array is just a struct that conforms to CustomStringConvertible, so that doesn't compile either. I could add an extension, but we should probably just improve SwiftLang's APIs / switch to whatever SourceKit-LSP is using to talk to sourcekitd.

Reply:
Ah, OK. Just leave it as is then.